doc/nvim-tree-lua.txt

```diff
@@ -548,12 +559,16 @@ Show LSP and COC diagnostics in the signcolumn
     Icons for diagnostic severity.
       Type: `table`, Default: `{ hint = "", info = "", warning = "", error = "" }`
 
-    `NOTE`: it will use the default diagnostic color groups to highlight the signs.
-    If you wish to customize, you can override these groups:
-    - `NvimTreeLspDiagnosticsError`
-    - `NvimTreeLspDiagnosticsWarning`
-    - `NvimTreeLspDiagnosticsInformation`
-    - `NvimTreeLspDiagnosticsHint`
+    *nvim-tree.diagnostics.severity*
+    Severity for which the diagnostics will be displayed. See |diagnostic-severity|
+
+      *nvim-tree.diagnostics.severity.min*
+      Minimum severity.
+        Type: |vim.diagnostic.severity|, Default: `vim.diagnostic.severity.HINT`
+
+      *nvim-tree.diagnostics.severity.max*
+      Minimum severity.
```

Comment:
```suggestion
      Maximum severity.
```

Reply:
Good catch, fixed.

FYI @gegoune you can usually directly push to forked repo branches. https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md#pull-request

I've been doing that for documentation and little nits like code style.